### ISSUE TITLE
Netstat input problem on CentOS 6 after upgrading to nightly release

### ISSUE THREAD

**Issue by sebastien-prudhomme** [influxdata/telegraf#3311]
### System info:

```
$ rpm -qi telegraf 
Name        : telegraf                     Relocations: / 
Version     : 1.5.0~b9f3195                     Vendor: InfluxData
Release     : 0                             Build Date: Thu 05 Oct 2017 11:21:40 AM CEST
```

Running on Centos 6 (Kernel 2.6.32-431.el6.x86_64)

### Actual behavior:

In telegraf.log:

```
2017-10-05T15:53:30Z E! Error in plugin [inputs.netstat]: error getting net connections info: cound not get pid(s), 0
```

### Additional info:

Works fine on previous installed version (telegraf-1.3.2-1.x86_64.rpm)

**Comment by danielnelson** [comment]
https://github.com/shirou/gopsutil/issues/433

**Comment by danielnelson** [comment]
I will update our gopsutil version on their next release.

**Comment by sebastien-prudhomme** [comment]
Thanks

**Comment by sebastien-prudhomme** [comment]
Seems there is a new release of gopsutils with the fix (v2.17.10)
